project/src/Level 0/Router.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { Route, Redirect, Switch } from 'react-router-dom';
+import Home from './Home'
+import Characters from './Characters'
+import Spells from './Spells'
+import Stories from './Stories'
+import HeadOfHouse from './HeadOfHouse'
+
+export default class Router extends Component {
+  render() {
+    return (
+      <div className='content'>
+        <Redirect
+          from='/'
+          to='/home'
+        />
+        <Switch>
+          <Route
+            path='/home'
+            component={Home}/>
```

Comment:
**Minor-Style**: Leave one blank space before the closing tag. Also, don't need to write in multiple lines when the amount of (key,values) are less than three.